### PR TITLE
Feature external web component

### PR DESCRIPTION
**Pull request by FelipeSimoes**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--genom--netcentric.hlx.page/webcomponent/hero
- After: https://feature-external-web-component--genom--netcentric.hlx.page/webcomponent/hero

### CODE REVIEW

File: blocks/columns/columns.js [hunk 1]
```diff
@@ -15,4 +15,5 @@ export default function decorate(block) {
       }
     });
   });
+  setupElement(block);
```
Comment (andreas-haller) [inline blocks/columns/columns.js]: this was only a test, right?

File: blocks/hero/hero.shadow.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  font-family: var(--heading-font-family);
+  font-weight: 600;
+  line-height: 1.25;
+  margin-top: 1em;
+  margin-bottom: 0.5em;
+  scroll-margin: calc(var(--nav-height) + 1em);
+}
+
+h1 {
+  font-size: var(--heading-font-size-xxl);
```
Comment (andreas-haller) [inline blocks/hero/hero.shadow.css:31]: wouldn't this be better inherited from the outside or is there no inheritance of styles?into the shadow dom?

**Review by easingthemes — APPROVED** [review]

File: libs/custom-params-component/custom-params-component.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
+export const breakpoints = {
+  s: 0,
+  m: 768,
+  l: 1024,
+  xl: 1280,
+  xxl: 1920,
```
Comment (andreas-haller) [inline libs/custom-params-component/custom-params-component.js]: for the viewports I think we should go with the Adobe "standard" of 600, 900, 1200, 1500

**Review by andreas-haller — APPROVED** [review]

File: blocks/hero/hero.shadow.css (hunk 2 above)
Comment (FelipeSimoes) [reply blocks/hero/hero.shadow.css:31]: shadow dom is encapsulated, it does inherit some things like body styles and variables so we need to style inside it

**Pull request merged**